Core/HLE/sceFont.cpp

```diff
@@ -511,7 +511,9 @@ class FontLib {
 		u32 args[2] = { params_.userDataAddr, (u32)handle_ };
 		// TODO: The return value of this is leaking.
 		if (handle_) {  // Avoid calling free-callback on double-free
-			__KernelDirectMipsCall(params_.freeFuncAddr, 0, args, 2, false);
+			if (coreState != CORE_POWERDOWN) {
```

Comment:
Why not?
```
if (handle_ && coreState != CORE_POWERDOWN) {
```

Reply:
Didn't really want to mess with or confuse the comments.

-[Unknown]

Reply:
I'm fine with either.